packages/paste-core/components/progress-steps/src/ProgressStepSeparator.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxProps} from '@twilio-paste/box';
+
+import {ProgressStepsContext} from './ProgressStepsContext';
+
+const VerticalStyles: BoxProps = {
+  borderLeftWidth: 'borderWidth20',
+  borderLeftStyle: 'solid',
+  borderLeftColor: 'colorBorderWeak',
+  minHeight: '60px',
+  borderRadius: 'borderRadius0',
+  marginLeft: 'space30',
+};
+const HorizontalStyles: BoxProps = {
```

Comment:
question: any risk to setting min width on the horizontal separator?

Reply:
Should be fine. I made it 32px

Reply:
nice